### ISSUE TITLE
Foreign key reference tables are blank

### ISSUE THREAD

**Issue by MarcelloTheArcane** [HeidiSQL/HeidiSQL#786]
## Steps to reproduce this issue

1. Step 1; Go to a table and select **Foreign keys**
2. Step 2; Add a key name and select a column
3. Step 3; Click the reference table dropdown
4. Then I get... nothing!

## Current behavior

No tables show in the dropdown:

![image](https://user-images.githubusercontent.com/21159570/67947179-f9914400-fbda-11e9-9ccd-7fc228239f43.png)

## Expected behavior

All the tables should be listed in the **Reference table** box for me to select.

## Environment

* **HeidiSQL version:** 10.2.0.5599
  
* **Database system and version:** PostgreSQL 10.5
  
* **Operating system:** Windows 10

**Comment by MarcelloTheArcane** [comment]
Yep, that was me.

**Comment by rwilliams** [comment]
Also already created foreign keys do not show up on postgres. Using postgres 11.5.3

**Comment by halmai** [comment]
The bug is still present with 

Version 10.3.0.5837 (64 Bit)
Compiled on: 2020-01-19 19:55:50

**Comment by ansgarbecker** [comment]
Could you please check with the latest build (5848 currently)

**Comment by rwilliams** [comment]
Foreign keys is still blank. build 5849 - psql (PostgreSQL) 12.1 (Debian 12.1-1.pgdg100+1).

![image](https://user-images.githubusercontent.com/29144/73126444-652f4700-3f67-11ea-85d4-a070a8667004.png)

**Comment by rwilliams** [comment]
Reference table does not populate and created foreign keys do not show. Build 5849. psql (PostgreSQL) 12.1 (Debian 12.1-1.pgdg100+1)

**Comment by ansgarbecker** [comment]
What would be a good SQL approach to retrieve foreign keys of one table? I know I can join `pg_index` with `pg_class` to get many index details, but for HeidiSQL that query should deliver also the reference table and columns from source and reference table:

```
SELECT * FROM pg_index i, pg_class c
WHERE
    i.indexrelid=c.oid
```

**Comment by rwilliams** [comment]
I found this on Stack overflow

```
SELECT
    tc.table_schema, 
    tc.constraint_name, 
    tc.table_name, 
    kcu.column_name, 
    ccu.table_schema AS foreign_table_schema,
    ccu.table_name AS foreign_table_name,
    ccu.column_name AS foreign_column_name 
FROM 
    information_schema.table_constraints AS tc 
    JOIN information_schema.key_column_usage AS kcu
      ON tc.constraint_name = kcu.constraint_name
      AND tc.table_schema = kcu.table_schema
    JOIN information_schema.constraint_column_usage AS ccu
      ON ccu.constraint_name = tc.constraint_name
      AND ccu.table_schema = tc.table_schema
WHERE tc.constraint_type = 'FOREIGN KEY' AND tc.table_name='truck_setups_workers';
```

![image](https://user-images.githubusercontent.com/29144/73140465-0d0f4800-402e-11ea-82eb-ee4e20162745.png)

**Comment by rwilliams** [comment]
In 5850 it looks like the reference tables are now populating.

**Comment by ansgarbecker** [comment]
> In 5850 it looks like the reference tables are now populating.

Really, in PostgreSQL? Or on MySQL or MS SQL?

**Comment by rwilliams** [comment]
Postgres

**Comment by ansgarbecker** [comment]
Oh yes, I was mixing this issue with #158 , which complains about non working detection of foreign keys. This one is just for the reference tables, which I fixed in conjunction with 7ab47a31434cda49e14604d7b2d2b1e79c89b5aa for #150 . 
Thanks for your feedback!
